ruby/vampires.rb

```diff
@@ -0,0 +1,84 @@
+#How many people are you going to be hired?
+puts "How many people are going to be hired?"
+hired = gets.chomp.to_i
+
+#Begin hired loop
+until hired == 0
+  # what's your name?
+  puts "What's your name?"
+  name = gets.chomp
+
+  #How old are you? What year were you born?
+  puts "How old are you?"
+  age = gets.chomp.to_i
+  puts "What year were you born?"
+  birth_year = gets.chomp.to_i
+  #Checking age in 2016
+  age_check = 2016 - birth_year - age
```

Comment:
Look at the time class in the Ruby docs to fine Time.now.year and see if you can find a dynamic way to add this to your code to make sure that we don't have to rewrite this file every year.